test/js/node/http/node-http.test.ts

```diff
@@ -2084,3 +2084,60 @@ it("ServerResponse ClientRequest field exposes agent getter", async () => {
     server.close();
   }
 });
+
+it("'st' package works", async () => {
```

Comment:
can we move this into a separate test file so it's clearer that its an integration test when it is flaky?